### PR TITLE
add securedrop-sdk 0.3.2 wheel

### PR DESCRIPTION
**Pull request by creviera**
## Description

* Adds `securedrop-sdk 0.3.2` wheel to our local wheels

### Test plan

- [ ] https://github.com/freedomofpress/securedrop-sdk/releases/tag/0.3.2 tag is verified and checked out in build logs: https://github.com/freedomofpress/build-logs/commit/e0772f8031d2137cf676331bc536a48afc5b1aa8#diff-7552bc13558e7a8280d315068f48b0216af29ff336c5bd6ae75a9c941fee0487R1-R28
- [ ] https://pypi.org/project/securedrop-sdk/0.3.2/#copy-hash-modal-405df297-24d0-4272-90d9-f13df413206a matches what's in build logs: https://github.com/freedomofpress/build-logs/commit/e0772f8031d2137cf676331bc536a48afc5b1aa8#diff-7552bc13558e7a8280d315068f48b0216af29ff336c5bd6ae75a9c941fee0487R70
- [ ] `sha256sums.txt` is signed by a SecureDrop maintainer
- [ ] Download source tarball from this PR and check that the hash matches what's in the build logs: https://github.com/freedomofpress/build-logs/commit/e0772f8031d2137cf676331bc536a48afc5b1aa8#diff-7552bc13558e7a8280d315068f48b0216af29ff336c5bd6ae75a9c941fee0487R4236
- [ ] Download wheel from this PR and check that the hash matches what's in the build logs: https://github.com/freedomofpress/build-logs/commit/e0772f8031d2137cf676331bc536a48afc5b1aa8#diff-7552bc13558e7a8280d315068f48b0216af29ff336c5bd6ae75a9c941fee0487R4238
- [ ] URL in the HTML part of the diff points to 0.3.2

### CODE REVIEW

**Comment by creviera** [comment]
After this is merged, https://github.com/freedomofpress/securedrop-client/pull/1298 is ready for a rerun of CI against the latest local wheels and review.

File: simple/securedrop-sdk/index.html [hunk 1]
```diff
@@ -21,5 +21,7 @@ <h1>Links for securdrop-sdk</h1>
     <a href="/localwheels/securedrop_sdk-0.3.0-py3-none-any.whl">securedrop_sdk-0.3.0-py3-none-any.whl</a><br/>
     <a href="/localwheels/securedrop-sdk-0.3.1.tar.gz">securedrop-sdk-0.3.1.tar.gz</a><br/>
     <a href="/localwheels/securedrop_sdk-0.3.1-py3-none-any.whl">securedrop_sdk-0.3.1-py3-none-any.whl</a><br/>
+    <a href="/localwheels/securedrop-sdk-0.3.1.tar.gz">securedrop-sdk-0.3.2.tar.gz</a><br/>
```
Comment (zenmonkeykstop) [inline simple/securedrop-sdk/index.html:24]: these should be updated to point to 0.3.2.

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]

File: simple/securedrop-sdk/index.html (hunk 1 above)
Comment (zenmonkeykstop) [inline simple/securedrop-sdk/index.html:24]: The tarball link also needs updating

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]

File: simple/securedrop-sdk/index.html (hunk 1 above)
Comment (creviera) [reply simple/securedrop-sdk/index.html]: twice in a row! sorry bout that. that's it i'm creating an automation ticket for this step

Comment (creviera) [reply simple/securedrop-sdk/index.html]: :boom: https://github.com/freedomofpress/securedrop-debian-packaging/issues/273

**Review by kushaldas — APPROVED** [review]
- [x] https://pypi.org/project/securedrop-sdk/0.3.2/#copy-hash-modal-405df297-24d0-4272-90d9-f13df413206a matches what's in build logs: https://github.com/freedomofpress/build-logs/commit/e0772f8031d2137cf676331bc536a48afc5b1aa8#diff-7552bc13558e7a8280d315068f48b0216af29ff336c5bd6ae75a9c941fee0487R70
- [x] `sha256sums.txt` is signed by a SecureDrop maintainer
- [x] Download source tarball from this PR and check that the hash matches what's in the build logs: https://github.com/freedomofpress/build-logs/commit/e0772f8031d2137cf676331bc536a48afc5b1aa8#diff-7552bc13558e7a8280d315068f48b0216af29ff336c5bd6ae75a9c941fee0487R4236
- [x] Download wheel from this PR and check that the hash matches what's in the build logs: https://github.com/freedomofpress/build-logs/commit/e0772f8031d2137cf676331bc536a48afc5b1aa8#diff-7552bc13558e7a8280d315068f48b0216af29ff336c5bd6ae75a9c941fee0487R4238
- [x] URL in the HTML part of the diff points to 0.3.2

All good. Approved.

**Comment by kushaldas** [comment]
I just now pushed a commit to remove all the old HTML files.

**Review by zenmonkeykstop — APPROVED** [review]

**Pull request merged**